src/App.js

```diff
@@ -82,20 +79,17 @@ class Board extends React.Component {
   //Check for conflict in col
   isPossibleCol(randomNumber, x, valueBoard) {
     for (let i = 0; i < 9; i++) {
-      //Change === to == to work
-      //console.log("valueBoard: " + valueBoard[i * 9 + x] + ", randomNumber: " + randomNumber); 
-      if(valueBoard[i * 9 + x] == randomNumber) {
-        console.log("True");
+      if(parseInt(valueBoard[i * 9 + x]) === randomNumber) {
         return false;
       }
     }
     return true;
   }
 
   //Checks 3 by 3 block
-  isPossibleBlock(randomNumber, boardSize, valueBoard) {
+  isPossibleBlock(randomNumber, position, valueBoard) {
     for (let i = 0; i < 9; i++) {
-      if (valueBoard[Math.floor(boardSize / 3) * 27 + i % 3 + 9 * Math.floor(i / 3) + 3 * (boardSize % 3)] == randomNumber) {
+      if (parseInt(valueBoard[Math.floor(position / 3) * 27 + i % 3 + 9 * Math.floor(i / 3) + 3 * (position % 3)]) === randomNumber) {
```

Comment:
This is hard to read/understand. Can you put pieces of it in variables with names to make it easier to see what's happening?

Reply:
Done!